modules/virt-running-real-time-checkup.adoc

```diff
@@ -0,0 +1,215 @@
+// Module included in the following assemblies:
+//
+// * virt/monitoring/virt-running-cluster-checkups.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="virt-running-real-time-checkup_{context}"]
+= Running a real-time checkup
+
+Use a predefined checkup to verify that your {product-title} cluster can run virtualized real-time workloads.
+
+
+.Prerequisites
+* You have installed the OpenShift CLI (`oc`).
+* The cluster is configured to run real-time applications.
```

Comment:
Due to tooling limitations, I cannot link to that topic directly from the `Prerequisites`, but if you notice I have added the link under an [Additional resources](https://75211--ocpdocs-pr.netlify.app/openshift-enterprise/latest/virt/monitoring/virt-running-cluster-checkups#virt-real-time-config-map-parameters_virt-running-cluster-checkups) heading right after the checkup procedure.

Reply:
Yes, I have noticed the `Additional resources` part (that's where I took the link from), I just thought it will be more effective to have it here as well. But if it's not possible then nevermind. I will resolve this comment then.